packages/docs/src/examples/app-bars/img-fade.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-card class="hide-overflow">
+    <v-app-bar
+      absolute
+      color="#6A76AB"
+      dark
+      shrink-on-scroll
+      prominent
+      src="https://picsum.photos/1920/1080?random"
+      fade-img-on-scroll
+      scroll-target="#scrolling-techniques-3"
+    >
+      <template #img="{ props }">
```

Comment:
https://github.com/vuetifyjs/vuetify/pull/6502#issuecomment-464061772

Reply:
Hrm, not sure about this one. We do the short hand for all features like `:value` instead of `v-bind:value` and `@click` instead of `v-on:click`. Other @vuetifyjs/core-team thoughts?

Reply:
True. But those shorthands have been around for quite a while now. I think since the slot syntax just recently changed, using `v-slot` might be better. At least for now.